CONTRIBUTING.rst

```diff
@@ -113,7 +113,7 @@ Before you submit a pull request, check that it meets these guidelines:
 1. The pull request should include tests.
 2. If the pull request adds functionality, the docs should be updated. Put
    your new functionality into a function with a `docstring`_.
-3. The pull request should work for Python 3.6, 3.7. and 3.8
+3. The pull request should work for Python 3.8
```

Comment:
"should work for Python 3.8, and higher (stable) versions of Python."

Reply:
but, agree, we can add the specific version numbers when we also officially have support for those future Python versions